docs/investigation/configuration/2022-06-14-policies.md

```diff
@@ -15,7 +15,7 @@ This section describes changes in the configuration syntax. As a result, we enab
       # Settings for Slack
       slack:
         token: 'SLACK_API_TOKEN'
-        notiftype: short
+        notifyType: short
```

Comment:
Let's change it to `notification.type` as well 👍  Same for other proposals / investigations.